assets/src/edit-story/components/library/panes/text/karma/textPane.karma.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../../karma/fixture';
+
+describe('CUJ: Creator can Add and Write Text: Consecutive text presets', () => {
+  let fixture;
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should add text presets below each other if added consecutively', async () => {
+    await fixture.editor.library.textTab.click();
+    await fixture.events.click(fixture.editor.library.text.preset('Heading 1'));
+    await fixture.events.click(fixture.editor.library.text.preset('Heading 3'));
+    await fixture.events.click(fixture.editor.library.text.preset('Paragraph'));
+
+    await fixture.snapshot('consecutively added different text presets');
+  });
+
+  it('should ensure staggered presets fit on the page', async () => {
+    await fixture.editor.library.textTab.click();
+    await fixture.events.click(
+      fixture.editor.library.text.preset('Heading 1'),
+      { clickCount: 10 }
+    );
+
+    await fixture.snapshot('staggered same type text presets');
```

Comment:
[This screenshot doesn't look right](https://percy.io/google/web-stories-wp/builds/6656538/view/381453371?browser=chrome&mode=head&width=1600) - it looks like all the text fields are added on top of each other. I think you need to await each click before proceeding.